Packs/HYASInsight/TestPlaybooks/Playbook-HYASInsight-Test.yml

```
@@ -0,0 +1,1292 @@
+id: Playbook-HYAS-Insight-Test
+version: 32
+vcShouldKeepItemLegacyProdMachine: false
+name: Playbook-HYASInsight-Test
```

Playbook name and ID should be the same.
```suggestion
name: Playbook-HYAS-Insight-Test
```